Example/AppDelegate.swift

```diff
@@ -15,6 +15,7 @@ final class AppDelegate: NSObject, NSApplicationDelegate {
 		let styles: [DockProgress.ProgressStyle] = [
 			.bar,
 			.circle(radius: 58, color: .systemPink),
+			.rectangle(width: 113, height: 113, x: 10, y: 10, color: .systemGreen),
```

Comment:
Put it above circle everywhere as `circle` is not very useful anymore with macOS 11.